app/controllers/users/reset_passwords_controller.rb

```diff
@@ -94,7 +94,7 @@ def handle_successful_password_reset
     end
 
     def handle_unsuccessful_password_reset(result)
-      if result.errors[:reset_password_token]
+      if result.errors[:reset_password_token].present?
```

Comment:
It was related to the Rails upgrade, but we should have done that before anyways since if `errors[:reset_password_token]` is an empty array, this will return `true`, which is not what we want.